### PR TITLE
Jr/integ 459 credential testing

### PR DESCRIPTION
**Pull request by johnathan-rodriguez**

### CODE REVIEW

File: shipyard_blueprints/dbt/shipyard_dbt/cli/authtest.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ def main():
     sys.exit(
         DbtClient(
             access_token=os.getenv("DBT_API_KEY"),
-            account_id=os.getenv("DBT_ACCOUNT_ID"),
+            account_id='',
```
Comment (wrp801) [inline shipyard_blueprints/dbt/shipyard_dbt/cli/authtest.py:11]: No change here, as we discussed

File: shipyard_blueprints/s3/shipyard_s3/cli/authtest.py [hunk 2]
```diff
@@ -12,8 +12,7 @@ def main():
     sys.exit(
         S3Client(
             aws_access_key=os.getenv("AWS_ACCESS_KEY_ID"),
-            aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
-            region=os.getenv("AWS_DEFAULT_REGION"),
```
Comment (wrp801) [inline shipyard_blueprints/s3/shipyard_s3/cli/authtest.py:16]: We need to leave the region as a required field here

**Review by wrp801 — CHANGES_REQUESTED** [review]
Main change is for S3, we need to have the region as a required credential field.

File: shipyard_blueprints/s3/shipyard_s3/cli/authtest.py (hunk 2 above)
Comment (johnathan-rodriguez) [reply shipyard_blueprints/s3/shipyard_s3/cli/authtest.py:16]: currently not being used in prod
image

**Review by wrp801 — APPROVED** [review]

**Pull request merged**